@kiva/kv-components/vue/KvRadio.vue

```diff
@@ -169,7 +169,7 @@ export default {
 		});
 
 		onMounted(() => {
-			uuid = `kvr-${nanoid(10)}`;
+			uuid.value = `kvr-${nanoid(10)}`;
```

Comment:
I think maybe the addition of `.value` is the culprit here.

Reply:
Adding `.value` is indeed the fix.